[2023-05-11T17:45:02.733+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:45:02.777+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor scheduled__2023-05-11T17:40:00+00:00 [queued]>
[2023-05-11T17:45:02.814+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor scheduled__2023-05-11T17:40:00+00:00 [queued]>
[2023-05-11T17:45:02.814+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:45:02.815+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-11T17:45:02.815+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:45:02.866+0000] {taskinstance.py:1383} INFO - Executing <Task(FileSensor): file_sensor> on 2023-05-11 17:40:00+00:00
[2023-05-11T17:45:02.878+0000] {standard_task_runner.py:54} INFO - Started process 334 to run task
[2023-05-11T17:45:02.888+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'job_1_tranvietcuong', 'file_sensor', 'scheduled__2023-05-11T17:40:00+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/tranvietcuong.py', '--cfg-path', '/tmp/tmpgvac0cub']
[2023-05-11T17:45:02.889+0000] {standard_task_runner.py:83} INFO - Job 162: Subtask file_sensor
[2023-05-11T17:45:02.892+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/tranvietcuong.py
[2023-05-11T17:45:03.212+0000] {task_command.py:384} INFO - Running <TaskInstance: job_1_tranvietcuong.file_sensor scheduled__2023-05-11T17:40:00+00:00 [running]> on host 09877495cfa0
[2023-05-11T17:45:03.226+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:873: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti = qry.one_or_none()

[2023-05-11T17:45:03.611+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=user@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=job_1_tranvietcuong
AIRFLOW_CTX_TASK_ID=file_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T17:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-11T17:40:00+00:00
[2023-05-11T17:45:03.686+0000] {base.py:71} INFO - Using connection ID 'fs_default' for task execution.
[2023-05-11T17:45:03.689+0000] {filesystem.py:58} INFO - Poking for file /opt/***/dags/tmp/*dejob*
[2023-05-11T17:45:03.692+0000] {filesystem.py:63} INFO - Found File /opt/***/dags/tmp/dejob.txt last modified: 20230511174143
[2023-05-11T17:45:03.693+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-05-11T17:45:03.709+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:45:03.729+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=job_1_tranvietcuong, task_id=file_sensor, execution_date=20230511T174000, start_date=20230511T174502, end_date=20230511T174503
[2023-05-11T17:45:03.796+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-11T17:45:03.964+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
